api/src/jvmMain/kotlin/Routes.kt

```diff
@@ -129,7 +126,10 @@ class Routes {
         class Risks(val build: Build)
 
         @Location("/tests-to-run")
-        class TestsToRun(val build: Build)
+        class TestsToRun(val build: Build) {
+            @Location("/duration")
+            class Duration(val testsToRun: TestsToRun)
```

Comment:
Should it be in the summary?

Reply:
in a new topic /build/summary/tests-to-run/duration ?
there are not suitable places in exist /build/summary